provider/resource_kpi_threshold_template.go

```diff
@@ -466,7 +466,11 @@ func (r *resourceKpiThresholdTemplate) Read(ctx context.Context, req resource.Re
 		return
 	}
 	if b == nil {
-		resp.Diagnostics.AddError("KPI threshold template is not found", fmt.Sprintf("KPI threshold template %s (%s) is not found", state.ID.ValueString(), state.Title.ValueString()))
```

Comment:
do we want to keep a warning, or are we ok with being totally silently adding a dummy template?

Reply:
Thank you, a warning, I added it for service, will update for others as well.

Reply:
Updated :)